packages-exp/auth-exp/src/core/strategies/credential.test.ts

```diff
@@ -86,8 +92,7 @@ describe('core/strategies/credential', () => {
         auth,
         authCredential
       );
-      expect(credential!.providerId).to.eq(ProviderId.FIREBASE);
```

Comment:
you should have the test ensure that the credential is preserved, maybe do UserCredential<MockAuthCredential> or whatevenr